node-typescript/sync-with-meilisearch/README.md

```diff
@@ -0,0 +1,106 @@
+# ⚡ Sync Appwrite to Meilisearch Function
```

Comment:
Can we change this to `Node.js (TypeScript) Sync Appwrite to Meilisearch Function`